Create README.md

RobillardA opened the pull request:
Added README.md file with Streamlit access button.

RobillardA committed:
Create README.md

Added README.md file with Streamlit access button.

MikeTrizna commented:
Hey @RobillardA, the path to the Streamlit app is actually https://share.streamlit.io/miketrizna/streamlit_fish_masking/main/fish_masking_classification_streamlit.py. We can definitely clean that up for the official deployment, but the link in your README won't work.

RobillardA commented:
I think that is the correct REAMME link. See the format for https://github.com/MikeTrizna/covid-streamlit/blob/main/README.md 
Unless I made a typo?

RobillardA committed:
README update

Adjusted the URL to point to the App directly (https://share.streamlit.io/miketrizna/streamlit_fish_masking/main/fish_masking_classification_streamlit.py) rather then "streamlit_app.py"

MikeTrizna commented:
Ok, that looks good now.
